rosdep/python.yaml

```diff
@@ -7101,6 +7101,18 @@ python3-networkx:
   fedora: [python3-networkx]
   nixos: [python3Packages.networkx]
   ubuntu: [python3-networkx]
+python3-nlopt:
+  arch: [nlopt]
+  debian:
+    '*': [python3-nlopt]
+    buster: null
+    stretch: null
+  fedora: [NLopt]
```

Comment:
I think this one should be `python3-NLopt`: https://packages.fedoraproject.org/pkgs/NLopt/python3-NLopt/ which comes from here: https://src.fedoraproject.org/rpms/NLopt/blob/rawhide/f/NLopt.spec#_149

Reply:
Hi, thanks for the catch! I uploaded the updated version now